app/styles/_data-toolbar.less

```diff
@@ -33,6 +33,12 @@
         font-size: @font-size-base;
       }
     }
+    .label-filter-key .selectize-input.not-full {
+      // Increase default input click target
+      width: 220px;
+      @media (min-width: @screen-xs) and (max-width: @screen-sm-max) { width: (@screen-xs - 100)} // 380px
+      @media (min-width: @screen-md) { width: (@screen-xs + 20)} // 500px
```

Comment:
I'm not sure how the width relates to `screen-xs` here. You set `max-width: 600px;` of the entire control above, so I'd think you would make width relative to that.